example_notebooks/16-tls_tree_dynamic.ipynb

```diff
@@ -0,0 +1,1091 @@
+{
```

Comment:
<p>I think "[...] an their attributed with laspy ..." should be "[...] an their attributes with laspy ...".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/3dgeo-heidelberg/helios/pull/369/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>